e2e/jest.test-setup.ts

```diff
@@ -36,35 +36,18 @@ afterEach(async () => {
  */
 beforeAll(async () => {
   await page.setRequestInterception(true);
-  page.on('request', (request) => {
-    const requestUrl = url.parse(request.url(), true);
-    const host = requestUrl.hostname;
-    // to improve page load performance, block network requests unrelated to application.
-    try {
-      if (host === 'www.google-analytics.com'
-           || host === 'accounts.youtube.com'
-           || host === 'static.zdassets.com'
-           || host === 'play.google.com'
-           || request.url().endsWith('content-security-index-report')) {
-        request.abort();
-      } else {
-        request.continue();
-      }
-    } catch (err) {
-      console.error(err);
-    }
```

Comment:
Remove because blocked requests will be logged as failed requests, making log harder to read.